test/test_rpc.ts

```diff
@@ -252,6 +252,44 @@ describe("Rpc", () => {
     assert.equal(after.callCount, 1);
   });
 
+  it("should support queueing messages when rpc is inactive", async () => {
+    const aRpc = new Rpc(defaults);
+    const bRpc = new Rpc(defaults);
+    // aRpc.start((msg) => bRpc.receiveMessage(msg));
+    bRpc.start((msg) => aRpc.receiveMessage(msg));
+
+    aRpc.registerImpl<IGreet>("greet", new MyGreeting(" from a"));
+    bRpc.registerImpl<IGreet>("greet", new MyGreeting(" from b"));
+
+    const bStub = bRpc.getStub<IGreet>("greet");
+    const aStub = aRpc.getStub<IGreet>("greet");
+
+    assert(bStub);
+    assert(aStub);
+    let bGreeting = bStub.getGreeting("Santa");
+    let aGreeting = aStub.getGreeting("Santa");
+
+    // wait only to make sure that both stubs are actually created without any errors, before we do
+    // actually start a rpc.
+    await new Promise((resolve) => setTimeout(resolve, 1));
```

Comment:
I don't see how this can help catch bugs (perhaps makes sense to do what you did below with noneResolved here too?)

Reply:
done